scripts/custom_lint_checks_test.py

```diff
@@ -38,12 +38,13 @@
 # pylint: enable=relative-import
 
 
-class ExplicitKwargsCheckerTest(unittest.TestCase):
+# TODO (apb7): Modify tests.
```

Comment:
In what way are you going to modify the tests? (That should be in your comment as well)

Reply:
Modified the tests and removed the TODO. Done, thanks!